packages/bot-web-ui/src/components/dashboard/bot-builder/bot-builder.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import RootStore from 'Stores/index';
+import AppStore from 'Stores/app-store';
+import { connect } from 'Stores/connect';
+import WorkspaceWrapper from './workspace-wrapper';
+
+type TBotBuilder = {
+    app: AppStore;
+};
+
+const BotBuilder = ({ app }: TBotBuilder) => {
+    const { onMount, onUnmount } = app;
+
+    React.useEffect(() => {
+        onMount();
+        return () => onUnmount();
+    }, []);
+
+    return (
+        <div
+            id='scratch_div'
+            style={{
```

Comment:
I'm not sure if using inline styles are ok in deriv-app or not, but it's better if we moved them out to some `scss` files.

Reply:
done ✅ (experimental not sure if it will work in all the cases)